tools/CodeGenerator/HttpProtocolFeatureCollection.cs

```diff
@@ -4,8 +4,6 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using Microsoft.AspNetCore.Http.Features;
-using Microsoft.AspNetCore.Http.Features.Authentication;
 //using Microsoft.AspNetCore.Server.Kestrel.Core.Features;
```

Comment:
Also unused?

Reply:
Will remove.  I didn't check for commented-out usings :).

Reply:
Good news is that was the only one